Topic4/src/classes/CashStrategy.java

```diff
@@ -0,0 +1,27 @@
+package classes;

```

Comment:
The package should have a more descriptive name not only classes, something related with the service that you are creating/modelling